dist/scripts/app.js

```diff
@@ -12,10 +12,17 @@
       url: '/',
       controller: 'RoomsCtrl as rooms',
       templateUrl: '/templates/home.html'
+    })
+    
+    //Do I need to inject the modal as a state? 
```

Comment:
You do not, because the modal isn't a new route in your app (it'll be open within an existing page, not when a new page is navigated to.